extractor/src/main/java/org/schabi/newpipe/extractor/services/media_ccc/extractors/MediaCCCRecentKiosk.java

```diff
@@ -40,7 +41,10 @@ public void onFetchPage(@Nonnull Downloader downloader) throws IOException, Extr
     @Override
     public InfoItemsPage<StreamInfoItem> getInitialPage() throws IOException, ExtractionException {
         final JsonArray events = doc.getArray("events");
-        StreamInfoItemsCollector collector = new StreamInfoItemsCollector(getServiceId());
+        Comparator<StreamInfoItem> comparator = Comparator.comparing(
+                streamInfoItem -> streamInfoItem.getUploadDate().offsetDateTime());
+        comparator = comparator.reversed();
+        StreamInfoItemsCollector collector = new StreamInfoItemsCollector(getServiceId(), comparator);
```

Comment:
For some reason, that does not work:
```suggestion
        StreamInfoItemsCollector collector = new StreamInfoItemsCollector(getServiceId(), Comparator.comparing(
                streamInfoItem -> streamInfoItem.getUploadDate().offsetDateTime()).reversed());
```